library/icons/html_code.html

```diff
@@ -0,0 +1,11 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/wp-content/themes/pps-2017/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/wp-content/themes/pps-2017/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/wp-content/themes/pps-2017/favicon-16x16.png">
+<link rel="manifest" href="/wp-content/themes/pps-2017/site.webmanifest">
+<link rel="mask-icon" href="/wp-content/themes/pps-2017/safari-pinned-tab.svg" color="#5bbad5">
+<link rel="shortcut icon" href="/wp-content/themes/pps-2017/favicon.ico">
+<meta name="apple-mobile-web-app-title" content="PPS.org">
+<meta name="application-name" content="PPS.org">
+<meta name="msapplication-TileColor" content="#4881cb">
+<meta name="msapplication-config" content="/wp-content/themes/pps-2017/browserconfig.xml">
+<meta name="theme-color" content="#d1dcf2">
```

Comment:
I think this is the third time we've seen this chunk of code?

Reply:
What code exactly?

Reply:
Oh oh oh… I could remove this file completely, actually. This is the same code that I already put into header.php. Its just a copy of the code that comes with the download package of icons. Sort of like when you download Icomoon and it included the sample CSS/HTML.